crates/biome_lsp_converters/src/lib.rs

```diff
@@ -148,50 +85,4 @@ mod tests {
         check_conversion!(line_index: Position { line: 0, character: 26 } => TextSize::from(32));
         check_conversion!(line_index: Position { line: 0, character: 27 } => TextSize::from(33));
     }
-
-    #[ignore]
-    #[test]
-    fn test_every_chars() {
-        let text: String = {
```

Comment:
This lives in `biome_line_index` because it specifically tests `LineIndex`, not an LSP related thing